examples/higher_transforms.ml

```diff
@@ -1,21 +1,20 @@
 open Joy.Shape
 
-(* Higher order transformations can be composed with `comp`, 
+(* Higher order transformations can be composed with `comp`,
    which applies its function args right-to-left.
    This allows us to create complex series transformations,
    that can be applied iteratively *)
 let transform = compose (translate 10 10) (scale 0.9)
-
 let rec range a b = if a > b then [] else a :: range (a + 1) b
 
-let () = 
+let () =
   init ();
-  let initial = rectangle ~x:(-250) ~y:(-250) 100 100 in 
-  let match_list l = 
-    match l with 
-    | [] -> [(transform initial)]
-    | last :: _ -> (transform last) :: l
-  in 
-  let shapes = List.fold_right (fun _ acc -> (match_list acc)) (range 0 32) [] in
+  let initial = rectangle ~x:(-250) ~y:(-250) 100 100 in
+  let match_list l =
+    match l with
+    | [] -> [ transform initial ]
+    | last :: _ -> transform last :: l
+  in
+  let shapes = List.fold_right (fun _ acc -> match_list acc) (range 0 32) [] in
   show shapes;
```

Comment:
i think this belongs in another PR and was merged already.

Reply:
I think this is here because I ran 'dune fmt'